Sources/MapboxCoreNavigation/EHorizon/EHorizon.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+import MapboxNavigationNative
+
+public class EHorizon {
```

Comment:
We should probably spell out “electronic” to make these types easier to find in code completion and less ambiguous when reading the code.